crates/hotshot/Cargo.toml

```diff
@@ -49,7 +49,9 @@ blake3.workspace = true
 url = { workspace = true }
 num_enum = "0.7"
 parking_lot = "0.12"
+twox-hash = { version = "1", default-features = false }
```

Comment:
Used to hash incoming messages and deduplicate them. Similar to how the combined network does it